qt/RpcClient.h

```diff
@@ -38,6 +38,7 @@ struct RpcResponse
 {
     QString result;
     TrVariantPtr args;
+    int64_t tag;
```

Comment:
Nit: is this still necessary? Appears to be unused now.

Reply:
0e7d9e1